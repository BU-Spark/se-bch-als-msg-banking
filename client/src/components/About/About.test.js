// This will be the test for About.js

// Imports
import React from "react";
import About from "./About";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";


// Tests:
// 1. Test that the About component renders
configure({adapter: new Adapter()});
test("About", () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
}
);
