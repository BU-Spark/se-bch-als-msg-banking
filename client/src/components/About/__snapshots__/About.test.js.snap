// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About 1`] = `ShallowWrapper {}`;
